contracts/src/challengeV2/EdgeChallengeManager.sol

```diff
@@ -257,6 +271,16 @@ contract EdgeChallengeManager is IEdgeChallengeManager, Initializable {
     /// @param stakeAmount  The amount of tokens being refunded
     event EdgeRefunded(bytes32 indexed edgeId, bytes32 indexed mutualId, address stakeToken, uint256 stakeAmount);
 
+    /// @notice Some stakes have been swept from defeated layer zero edges
+    /// @param defeatedEdgeIds  The ids of the defeated edges
+    /// @param stakeToken       The ERC20 being swept
+    /// @param totalStake       The total amount of tokens being swept
+    event DefeatedEdgeStakesSwept(
+        bytes32[] defeatedEdgeIds,
```

Comment:
I think its better to emit each edge indexed, maybe also with the mutual id it associated with can be nicer